app/components/badge.js

```diff
@@ -104,17 +104,21 @@ export default class Badge extends PureComponent {
                     style={[styles.text, this.props.countStyle]}
                     onLayout={this.onLayout}
                 >
-                    <View style={[styles.unreadIndicator, {backgroundColor: this.props.countStyle.color}]}/>
+                    <View style={styles.verticalAlign}>
+                        <View style={[styles.unreadIndicator, {backgroundColor: this.props.countStyle.color}]}/>
+                    </View>
                 </View>
             );
         } else {
             unreadCount = (
-                <Text
-                    style={[styles.text, this.props.countStyle]}
-                    onLayout={this.onLayout}
-                >
-                    {count.toString()}
-                </Text>
+                <View style={styles.verticalAlign}>
```

Comment:
Unread count on android tablet seems to be off by 0.5px or so to the right as seen in the screenshot. I tried fixing and spent fair share of time on this already. 
I will let QA test this on a real device as it could be an emulator issue and have them create a follow ticket if needed.